packages/frontend/app/helpers/year-next.js

```diff
@@ -0,0 +1,7 @@
+import ENV from 'frontend/config/environment';
+
+export default function yearNext(year) {
+  const yearNext = (parseInt(year) + 1).toString();
+
+  return ENV.environment != 'production' ? yearNext.substring(2) : yearNext;
```

Comment:
I'm not sure why this would change based on the environment, can you add a comment explaining?

Reply:
This was done in response to this: https://github.com/ilios/ilios/issues/3422#issuecomment-2083248287